spec/widgets/histogram/content-view.spec.js

```diff
@@ -12,7 +12,8 @@ describe('widgets/histogram/content-view', function () {
     });
 
     this.widgetModel = new HistogramWidgetModel({
-      title: 'Howdy'
+      title: 'Howdy',
+      attrsNames: ['title']
```

Comment:
But we don't need that here, attrsNames of the proper HistogramWidgetModel should contain the title, shouldn't it?